packages/core/admin/server/tests/admin-api-token.test.e2e.js

```diff
@@ -608,4 +608,22 @@ describe('Admin API Token v2 CRUD (e2e)', () => {
       updatedAt: expect.any(String),
     });
   });
+
+  test('Regenerates an api token access key', async () => {
+    const token = await createValidToken();
+
+    const res = await rq({
+      url: `/admin/api-tokens/${token.id}/regenerate`,
+      method: 'POST',
+    });
+
+    expect(res.statusCode).toBe(201);
+    expect(res.body.data).toMatchObject({
```

Comment:
We could also check that the accessKey is different from `token.accessKey`, here it would still pass if we break the regeneration and return the same accessKey.

Reply:
Done!